src/Components/Components/src/Rendering/HtmlRenderer.cs

```diff
@@ -43,12 +44,12 @@ protected override Task UpdateDisplayAsync(in RenderBatch renderBatch)
         /// Renders a component into a sequence of <see cref="string"/> fragments that represent the textual representation
         /// of the HTML produced by the component.
         /// </summary>
-        /// <typeparam name="T">The type of the <see cref="IComponent"/>.</typeparam>
+        /// <typeparam name="TComponent">The type of the <see cref="IComponent"/>.</typeparam>
         /// <param name="initialParameters">A <see cref="ParameterCollection"/> with the initial parameters to render the component.</param>
         /// <returns>A sequence of <see cref="string"/> fragments that represent the HTML text of the component.</returns>
-        public IEnumerable<string> RenderComponent<T>(ParameterCollection initialParameters) where T : IComponent
```

Comment:
I'm all for killing the sync versions of these. @SteveSandersonMS any objection? (We can file a separate issue for this and do it in one go)

Reply:
Totally agreed.